package.json

```diff
@@ -56,7 +56,7 @@
     "querystring-es3": "^0.2.1",
     "stream-browserify": "^3.0.0",
     "ts-node": "^10.9.1",
-    "typescript": "5.5.4",
+    "typescript": "~5.6",
```

Comment:
`~` gets the latest patch, but not the next minor version. I can also just specify the exact version number instead of providing a range.

Reply:
Yup, this is perfectly fine.